### ISSUE TITLE
hl.exe or csstrike.exe not found/running!

### ISSUE THREAD

**Issue by qvimxyz** [eVOL-UC/cs-1.6#2]
![image](https://user-images.githubusercontent.com/44214566/120901300-530bf100-c610-11eb-841b-e1cb1127522e.png)
idk what is the issue

**Comment by eVOL-UC** [comment]
i dont know what the issue could be, is this gsclient?

**Comment by qvimxyz** [comment]
> 
> 
> i dont know what the issue could be, is this gsclient?
its not gc client
im not using original cs and even when i try to inject the dll with another injector the game crashes

**Comment by qvimxyz** [comment]
so i tried with GSclient and i got this 

![image](https://user-images.githubusercontent.com/44214566/121080185-7cb04e00-c7b1-11eb-83b4-1d38cde677e0.png)

**Comment by eVOL-UC** [comment]
yes it doesnt work for gsclient, thats why i asked. in the first photo, your viewing the application information saying "Half-Life Launcher", if you view the process information, does it show hl.exe or cstrike.exe running? i can only suggest you to buy steam cs 1.6, it should work. i know windows 10 has some issues that i havent been able to solve because i never upgraded from windows 7. if you have any antivirus then maybe it will help to disable it,
 if you do then you need to charge the folder name when trying again.

its strange that you have this white line at the bottom of injector
![Untitled](https://user-images.githubusercontent.com/72867961/121097314-1862ae00-c7f4-11eb-8ef5-01cbc380195f.png)

**Comment by eVOL-UC** [comment]
if the process is cs.exe then you can change it here to look for it
![Untitled](https://user-images.githubusercontent.com/72867961/121097995-3da3ec00-c7f5-11eb-99f8-af2685f3e7ef.png)

**Comment by qvimxyz** [comment]
1. I don't know anything about this white line since I can remember it was always like this
2.When I open the game without admin and use the loader the game crashes
3. when I open the game with admin, the loader gives an error of "not found"
4. steam cs works but i don't want to risk taking a ban

![Screenshot_5](https://user-images.githubusercontent.com/44214566/121106035-9797b800-c7db-11eb-8a99-5a9f7f111315.png)
hl.exe

After some testing I managed to use it, I used the guidedhacking loader and managed to inject without crashing, maybe it's the loader? I don't know but it's working here again.

![image](https://user-images.githubusercontent.com/44214566/121106139-d299eb80-c7db-11eb-900d-e4562fcbacbc.png)
